twine/__main__.py

```diff
@@ -15,26 +15,19 @@
 import sys
 from typing import Any
 
+import colorama
 import requests
 
 from twine import cli
 from twine import exceptions
 
-try:
-    import colorama
-except Exception:
-    colorama = None
-
 
 def main() -> Any:
     try:
-        if colorama:
-            colorama.init()
+        colorama.init()
         return cli.dispatch(sys.argv[1:])
     except (exceptions.TwineException, requests.HTTPError) as exc:
-        pre_style, post_style = "", ""
-        if colorama:
-            pre_style, post_style = colorama.Fore.RED, colorama.Style.RESET_ALL
+        pre_style, post_style = colorama.Fore.RED, colorama.Style.RESET_ALL
```

Comment:
Since we're not conditionally retrieving these, why are we keeping these local bindings?

Reply:
Resolved in next commit